tests/3d_examples/test_3d_dambreak/CMakeLists.txt

```diff
@@ -34,8 +34,8 @@ file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/regression_test_tool/Fluidobserver_Pressur
 aux_source_directory(. DIR_SRCS)
 ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS})
 
-#add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME}
-#                   WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME}
```

Comment:
you can also do so:
add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME}  --rt=false

so that you just deactivate the regression test of this test